### DISCUSSION TITLE
[New Feature] Visual display for unit / structure under attack

### DISCUSSION THREAD

**Discussion by Rampastring** [Vinifera-Developers/Vinifera#1511] [Feature Requests]
### Description:
TS lacks some convenience features that more modern games tend to have for making it clear what's going on. For example, it's not very rare to miss a sneak attack on your base because you were focused on the offense. You could argue that it's a skill to keep an attention on things and I generally agree, but there already is the EVA "base under attack" notification, so clearly you are meant to be signaled about bad things happening to your stuff. The current notifications just don't seem to often be enough.

I think it'd be nice if we could add some pointers to the screen on where your objects are getting attacked. Say, if your units or buildings are getting attacked to the right of your viewport, the game could draw some kind of a red arrow to the right edge of the screen to symbol this.

This behaviour should naturally be toggleable through a Rules.ini key, and possibly also in SUN.ini.

**Comment by Crimsonum** [comment]
Good idea. A simple flashing red light/button accompanied by a warning sound should also be sufficient, like in Generals.

I'm not sure if it's a problem in vanilla too, but in TI/patched TS the base attack event doesn't even play most of the time, which can be extremely frustrating.
